### PR TITLE
adds ability to add users and points to tasks

### PR DESCRIPTION
**Pull request by TrevorFlanigan**

### CODE REVIEW

**Comment by TrevorFlanigan** [comment]
This cannot be merged just yet - need to test some more edge cases

**Review by armintalaie — APPROVED** [review]

**Comment by TrevorFlanigan** [comment]
Ready to merge

File: frontend/lib/src/authentication_home_page.dart [hunk 1]
```diff
@@ -24,7 +36,7 @@ class LoginPage extends StatelessWidget {
       );
       // Once signed in, return the UserCredential
       final ret = await FirebaseAuth.instance.signInWithCredential(credential);
-      FirebaseFirestore firestore = FirebaseFirestore.instance;
+      // FirebaseFirestore firestore = FirebaseFirestore.instance;
```
Comment (gokcedilek) [inline frontend/lib/src/authentication_home_page.dart:39]: could we remove the commented part?

File: frontend/lib/src/task.dart [hunk 2]
```diff
@@ -76,14 +78,28 @@ class _Task extends State<Task> {
...
+                    alignment: Alignment.centerLeft,
+                    child: Text(
+                      widget.description,
+                    ),
+                  ),
+                  Align(
+                    alignment: Alignment.centerLeft,
+                    child: Text(
+                      "Points: $points",
+                      style: const TextStyle(
```
Comment (gokcedilek) [inline frontend/lib/src/task.dart:96]: not specifically for this PR, but for the future, is it possible for us to look into fetching these styles from a common place, such as something like `Styles.RegularText` that contains all these css?

**Review by gokcedilek — APPROVED** [review]
